.env.example

```diff
@@ -115,3 +121,23 @@ AIRFLOW_SERVICE_SCHEME=http
 AIRFLOW_SERVICE_HOST=airflow-webserver
 AIRFLOW_SERVICE_PORT=8080
 AIRFLOW_SERVICE_PATH_PREFIX=/api/v1
+
+# Search Service
+ES_HOST=badgerdoc-elasticsearch
+ES_PORT=9200
+APP_TITLE=Badgerdoc Search
+ANNOTATION_URL=http://annotation
+ANNOTATION_CATEGORIES=/categories
+ANNOTATION_CATEGORIES_SEARCH=/categories/search
+MANIFEST=manifest.json
+TEXT_PIECES_PATH=/pieces
+INDEXATION_PATH=/indexation
+JOBS_URL=http://jobs
+JOBS_SEARCH=/jobs/search
+COMPUTED_FIELDS=["job_id", "category"]
+EMBED_URL=http://embeddings:3334/api/use
```

Comment:
If EMBED is new microservice, ensure configuration aligned with other host configuration, for example:
```
EMBEDINGS_SERVICE_SCHEME=http
EMBEDINGS_SERVICE_HOST=badgerdoc-embeddings
EMBEDINGS_SERVICE_PORT=8080
```